Assets/MixedRealityToolkit/_Core/Definitions/Devices/MixedRealityInteractionMapping.cs

```diff
@@ -223,14 +257,23 @@ public void SetValue(SixDof newValue)
                 Debug.LogError("SetValue(SixDof) is only valid for AxisType.SixDoF InteractionMappings");
             }
 
-            Changed = transformData.Position != newValue.Position ||
-                      transformData.Rotation != newValue.Rotation;
+            Changed = (sixdofData != null) ? !sixdofData.Equals(newValue) : (newValue != null);
```

Comment:
SixDof is a struct and can never be null.

Reply:
So, what do you want the changed statement to be?

Reply:
See the PR I just opened. It addresses that and some other things.